SU-SOE/stanford_soe_helper
# 7.x 2.x soe 3038

# READY FOR REVIEW

# Summary
This adds the Injector file 50 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 50 into SCSS


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-3038 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-3038)
- edit the injector: `admin/config/development/css-injector/edit/50` and disable it.
- `drush cc css-js`
- go to: `magazine/article/jeremy-bailenson-taking-grand-tour-latest-virtual-reality` and make sure the changes look good.

# Injector code and screen shots
`.dm-global-header #site-title-first-line.site-title-uppercase,
.dm-global-header #site-title-first-line.site-title-uppercase a {
    font-size: 18px;
}`

![screen shot 2018-06-19 at 8 51 34 am](https://user-images.githubusercontent.com/571421/41601455-229cdd1c-739e-11e8-9eea-75eca0b0a380.png)

`a {
    font-weight: 400;
}`

![screen shot 2018-06-19 at 9 08 46 am](https://user-images.githubusercontent.com/571421/41602883-6649cd10-73a1-11e8-95a6-26f3dca773b4.png)

`.view-stanford-magazine-article-mag-landing-page .mag-article-container .mag-article-content-container .mag-article-title a, 
.view-stanford-magazine-collection-mag-landing-page .mag-article-container .mag-article-content-container .mag-article-title a, 
.view-display-id-article_collection_bottom_block .mag-article-container .mag-article-content-container .mag-article-title a {
  font-weight: 600;
}`

![screen shot 2018-06-19 at 9 19 35 am](https://user-images.githubusercontent.com/571421/41603126-f54f837e-73a1-11e8-95ca-1366e98c2856.png)

`.view-stanford-magazine-article-most-recent.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-articles.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-topics.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-issues .mag-article-container .mag-article-content-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-issues .mag-feat-article-container .mag-article-content-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-article-most-recent.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-articles.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-topics.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a{
  font-weight: 600;
}`
![screen shot 2018-06-18 at 5 35 03 pm](https://user-images.githubusercontent.com/571421/41565831-09e49722-731e-11e8-888f-b99d6d5c581a.png)

`#digital-magazine-menu .region-digital-magazine-menu .block-menu-block ul li a {
  font-weight: 600;
}`
![screen shot 2018-06-18 at 5 38 54 pm](https://user-images.githubusercontent.com/571421/41565911-88c4a186-731e-11e8-9761-8a3d1c8185fd.png)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3038

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
